src/qt/forms/tpospage.ui

```diff
@@ -174,7 +174,7 @@
               <string> %</string>
              </property>
              <property name="minimum">
-              <number>1</number>
+              <number>0</number>
```

Comment:
Are you sure there are no further validations about this on the protocol itself?